platforms/core-runtime/launcher/src/integTest/groovy/org/gradle/launcher/SystemClassLoaderTest.groovy

```diff
@@ -48,8 +48,6 @@ class SystemClassLoaderTest extends AbstractIntegrationSpec {
                 doLast {
                     def systemLoader = ClassLoader.systemClassLoader
 
-                    systemLoader.loadClass("org.gradle.launcher.daemon.bootstrap.GradleDaemon") // this should be on the classpath, it's from the launcher package
```

Comment:
The point of this test is to verify that the Gradle implementation is not present in the system ClassLoader, but this assertion is checking that the daemon implementation is present, which is kind of pointless because otherwise the test won't make it this far.